dm/tests/all_mode/run.sh

```diff
@@ -531,7 +587,6 @@ function run() {
 	run_sql_both_source "SET @@global.time_zone = 'SYSTEM';"
 }
 
-cleanup_data all_mode
```

Comment:
this is still needed, because your newly added `test_regexpr_router` is not the first case.